docs/source/knowledgebase/glossary.rst

```diff
@@ -48,7 +48,7 @@ Glossary
 
    Local Average Treatment effect
    LATE
-      Also known asthe complier average causal effect (CACE), is the effect of a treatment for subjects who comply with the experimental treatment assigned to their sample group. It is the quantity we're estimating in IV designs.
```

Comment:
On this one, I think "asthe" is incorrect, but "complier" is actually correct. Might need to add "complier" to the whitelist and manually change "asthe" to "as the". Confusing why it's not flagged "asthe".

Reply:
Yeah that's weird but should be handled by the dictionary